src/openrct2/actions/LargeSceneryRemoveAction.cpp

```diff
@@ -65,15 +65,18 @@ GameActions::Result LargeSceneryRemoveAction::Query() const
     TileElement* tileElement = FindLargeSceneryElement(_loc, _tileIndex);
     if (tileElement == nullptr)
     {
-        LOG_WARNING("Invalid game command for scenery removal, x = %d, y = %d", _loc.x, _loc.y);
+        LOG_ERROR("No large scenery element to remove at x = %d, y = %d", _loc.x, _loc.y);
         return GameActions::Result(
             GameActions::Status::InvalidParameters, STR_CANT_REMOVE_THIS, STR_INVALID_SELECTION_OF_OBJECTS);
     }
 
     auto* sceneryEntry = tileElement->AsLargeScenery()->GetEntry();
     // If we have a bugged scenery entry, do not touch the tile element.
     if (sceneryEntry == nullptr)
-        return GameActions::Result(GameActions::Status::Unknown, STR_CANT_REMOVE_THIS, STR_UNKNOWN_OBJECT_TYPE);
+    {
+        LOG_WARNING("Scenery entry at x = %d, y = %d not removed because it is an unknown object type", _loc.x, _loc.y);
+    }
+    return GameActions::Result(GameActions::Status::Unknown, STR_CANT_REMOVE_THIS, STR_UNKNOWN_OBJECT_TYPE);
```

Comment:
Should have been inside the `if`.

Reply:
Shouldn't this trigger some unreachable code warning in CI? I'm quite surprised this passed the automated checks.

Reply:
Me too - compilation already fails if you define a variable that you don’t use, so I’m surprised it does allow unreachable code.